### PR TITLE
Icons & Button

### PR DESCRIPTION
**Pull request by JFKFred**
* 37x Icons
* 11 Button Variants

### CODE REVIEW

File: client/src/components/ui/button.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      },
+    },
+    defaultVariants: {
+      variant: "default",
+      size: "default",
+    },
+  },
+);
+
+const commonStyle =
```
Comment (pChausseC) [inline client/src/components/ui/button.tsx]: `commonStyle` is only used once. from its name I understand that it should be reused in the button variants? if not it doenst have to exist, you can inline the classname on line 112

File: client/src/components/h.tsx
Comment (pChausseC) [inline client/src/components/h.tsx:1]: maybe this is out of scope for a pr named "Icons & Buttons" but I think this component should handle all text tags, not just hX (and be name something like ```Typography```)

**Review by pChausseC — CHANGES_REQUESTED** [review]

File: client/src/components/h.tsx
Comment (JFKFred) [reply client/src/components/h.tsx:1]: Yeah, I just throw some quick thing for title. My PR will for this components then.

File: client/src/components/ui/button.tsx (hunk 1 above)
Comment (JFKFred) [reply client/src/components/ui/button.tsx]: removed

**Review by pChausseC — APPROVED** [review]

**Pull request merged**